src/runtimes/node/parser/imports.ts

```diff
@@ -0,0 +1,81 @@
+import { Statement } from '@babel/types'
+
+import { nonNullable } from '../../../utils/non_nullable'
+
+import { isImport, isRequire } from './helpers'
+
+// Finds import/require statements of a given path in an AST.
+const getImports = (node: Statement, importPath: string) => {
+  const esmImports = getImportsFromESM(node, importPath)
+
+  if (esmImports.length !== 0) {
+    return esmImports
+  }
+
+  const cjsImports = getImportsFromCJS(node, importPath)
+
+  return cjsImports
+}
+
+// Finds import/require statements of a given path in a CJS AST.
+const getImportsFromCJS = (node: Statement, importPath: string) => {
+  if (node.type !== 'VariableDeclaration') {
+    return []
+  }
+
+  const { declarations } = node
+  const requireDeclaration = declarations.find(
+    (declaration) => declaration.type === 'VariableDeclarator' && isRequire(declaration.init, importPath),
+  )
+
+  if (requireDeclaration === undefined || requireDeclaration.id.type !== 'ObjectPattern') {
```

Comment:
Testing for `ObjectPattern` will mean we miss this:

```js
const nfFunctions = require("@netlify/functions")

exports.handler = nfFunctions.cron(...)
```

Probably out-of-scope for this PR, though.

Reply:
Yeah, I think we can expand the patterns we catch in subsequent PRs, probably after the Beta release.